doc/how_to/custom_components/esm/custom_layout.md

```diff
@@ -114,114 +98,125 @@ import panel as pn
 
 from panel.custom import Child, ReactComponent
 
-pn.extension()
+CSS = """
+.split {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+    width: 100%;
+}
 
-class SingleObjectLayout(ReactComponent):
-    object = Child(allow_refs=False)
+.gutter {
+    background-color: #eee;
+    background-repeat: no-repeat;
+    background-position: 50%;
+}
 
-    _esm = """
-export function render({ model }) {
-    return (
-        <div>
-            <h1>Temperature</h1>
-            <h2>A measurement from the sensor</h2>
-            <div>
-                {model.get_child("object")}
-            </div>
-        </div>
-    );
+.gutter.gutter-horizontal {
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
+    cursor: col-resize;
 }
 """
 
-dial = pn.widgets.Dial(
-    name="°C",
-    value=37,
-    format="{value}",
-    colors=[(0.40, "green"), (1, "red")],
-    bounds=(0, 100),
-)
-react_layout = SingleObjectLayout(
-    object=dial,
-    name="Temperature",
-    styles={"border": "2px solid lightgray"},
+
+class SplitReact(ReactComponent):
+
+    left = Child()
+    right = Child()
+
+    _esm = """
+    import Split from 'https://esm.sh/react-split@2.0.14'
+
+    export function render({ model }) {
+        return (
+            <Split className="split">
+                <div>{model.get_child("left")}</div>
+                <div>{model.get_child("right")}</div>
```

Comment:
Not need for additional nesting, in fact this sizes correctly for me while the previous version did not:

```suggestion
                {model.get_child("left")}
                {model.get_child("right")}
```

Reply:
Could it be a browser difference? I use Edge.

Reply:
Was an issue with dev mode inserting an extra element that messed with the layout. Now fixed.